nunit.tests.csharp/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("nunit.tests")]
-[assembly: AssemblyCopyright("Copyright © 2017")]
+[assembly: AssemblyCopyright("Copyright © 2019")]
```

Comment:
I'd be inclined to remove the AssemblyInfo for a test project template.

Reply:
Done